test/orchard/namespace_test.clj

```diff
@@ -62,6 +62,9 @@
       (testing "when multiple ns forms are present"
         (spit url "(ns ns1) (ns ns2) (ns ns3)")
         (is (= 'ns1 (sut/read-namespace uri))))
+      (testing "when ns form is invalid"
+        (spit url "(ns (:require [clojure.string]))")
```

Comment:
`(ns foo (:requires [clojure.string]))` is a also a good example to be covered.

Accordingly, probably a try/catch would be better than the proposed approach - there can be few reasons why a ns parsing could fail.

Reply:
The error I've reported in #158 does not occur here, but it originates here, see:

https://github.com/clojure-emacs/orchard/blob/c0e24f71292f4fed8d4d8cf854a99e4a80cf6ee6/src/orchard/namespace.clj#L121-L123

What happens without my change is that we try to sort and hence compare a seq like `(:requires [clojure.string])` with symbols and that fails.

I do think a try/catch could be good & helpful in addition to that, but then I'd place it way further up the stack, possibly in cider-nrepl in `analyze-stacktrace` https://github.com/clojure-emacs/cider-nrepl/blob/5c0f21197fcccb1b2ca67054cab1dcc8a6af2c7f/src/cider/nrepl/middleware/stacktrace.clj#L216

Btw, re-reading the issue I'm noticing now I should have pointed out the stacktrace in https://gist.github.com/mk/bd82092048d733ef3de0e312898f47b4 more promintely.